validator/templates/validator/help.html

```diff
@@ -71,7 +71,21 @@ <h1 class="h3 mb-3 font-weight-normal">Validate Page</h1>
     <p>
         <b>Step 5 [optional]</b> - Choose the criteria by which you would like to filter the reference data prior to
         running the validation. The filters available depend on the data contained within the chosen dataset. For example,
-        you can filter the ISMN data to include only data points where the soil_moisture_flag is &quot;G&quot; for &quot;good&quot;.
+        you can filter the ISMN data to include only data points where the soil_moisture_flag is &quot;G&quot; for &quot;good&quot;. You can also choose particular
+        networks and depth range at which the measurements have been done. To do that you have to open an appropriate window by clicking a 'select...'
```

Comment:
Could you use: '...also choose specific networks and/or measurements in a defined depth range'